drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/users/show.haml

```diff
@@ -0,0 +1,24 @@
+- content_for :modal_title, "Full Permissions for #{@user.name}"
+
+%h1= @user.name
```

Comment:
Since name is in the header, I think we can leave it off here.

Reply:
✅